### PR TITLE
Fix & optimization

### PR DESCRIPTION
**Pull request by Xaliks**
image

### CODE REVIEW

File: src/interactions/levels/Leaderboard.ts [hunk 1]
```diff
@@ -19,29 +19,32 @@ export default class GiveXP extends SubCommand {
...
+        embed.addField(
+          member.user.username,
+          `${isInPlace ? places[idx] : ""} ${this.bot.utils.formatNumber(user.xp)}xp`,
+          true,
+        );
+
+        ++idx;
+      }
+
+      if (idx === data.slice(0, 10).length - 1) break;
```
Comment (casperiv0) [inline src/interactions/levels/Leaderboard.ts:46]: A `for` loop may be even better here? `for (let i = 0; i < 10; i++)`

**Review by casperiv0 — CHANGES_REQUESTED** [review]
See comments ;)

File: src/interactions/levels/Leaderboard.ts (hunk 1 above)
Comment (Xaliks) [reply src/interactions/levels/Leaderboard.ts:46]: `for (let i = 0; i < 10; i++)` ~ 0.9sec -> https://i.xaliks.xyz/oseJ.png
`for (const user of data)` ~ 0.85sec -> https://i.xaliks.xyz/bnGN.png

Comment (casperiv0) [reply src/interactions/levels/Leaderboard.ts:46]: It's only a few milliseconds, I'd rather have readability over speed here :)

**Comment by Xaliks** [comment]
See comments ;)

**Pull request merged**